corehq/warehouse/transforms/sql/app_status_form_staging.sql

```diff
@@ -0,0 +1,50 @@
+INSERT INTO {{ app_status_form_staging }} (
+       last_submission,
+       submission_build_version,
+       commcare_version,
+       user_id,
+       app_id,
+       domain,
+       batch_id
+)
+
+SELECT
+    last_submission,
+    submission_build_version,
+    commcare_version,
+    user_id,
+    app_id,
+    domain,
+    '{{ batch_id }}'
+FROM
+(
+SELECT
+	row_number() over wnd,
+	CASE
+	    WHEN received_on < app_status.last_form_submission_date
+	    THEN app_status.last_form_submission_date
+	    ELSE  first_value(received_on AT TIME ZONE 'UTC') OVER wnd
```

Comment:
(same question here, the first received_on would be the oldest one since we're ordering by it)

Reply:
sorry nevermind. i see it's ordering descending

Reply:
actually it occurs to me that we probably can get rid of `first_value` here since we're only taking the first row in the window later on anyway

Reply:
good point, i will get rid of that function call